staging/cse/windows/nvidiagpudriverfunc.ps1

```diff
@@ -0,0 +1,178 @@
+function Start-InstallGPUDriver {
+    param(
+        [Parameter(Mandatory = $true)]
+        [bool]$EnableInstall,
+        [Parameter(Mandatory = $true)]
+        [PSCustomObject]$DriverConfig
```

Comment:
Can we use string URL instead of object as the parameter? It will help us to reduce the cse length in https://github.com/Azure/AgentBaker/pull/3820/files#diff-fdbab787063757b4c74fc431a241ee84d3f02448e4f60b4b22f731c917832fbb